ocaml/testsuite/tests/typing-layouts/basics_alpha.ml

```diff
@@ -335,9 +336,9 @@ let id5 : 'a void5 -> 'a void5 = function
 [%%expect{|
 type ('a : void) void5 = Void5 of 'a
 type 'a any5 = Any5 of 'a
-Lines 4-5, characters 33-22:
-4 | .................................function
+Line 5, characters 15-22:
 5 |   | Void5 x -> Void5 x
+                   ^^^^^^^
```

Comment:
See https://github.com/ocaml-flambda/flambda-backend/pull/1817/files/c50a41a9351da707453169d6089b8803d94b1e42#r1363687731; this will have the same resolution as that.